tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_access_opened_files.py

```diff
@@ -0,0 +1,249 @@
+"""
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Check that files that are being scanned by the syscheckd daemon can
+       modified (renamed/deleted), and that wazuh is not blocking the files.
+
+tier: 1
+
+modules:
+    - syscheck
+
+components:
+    - manager
+    - agent
+
+path: tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_access_opened_files.py
+
+daemons:
+    - wazuh-syscheckd
+
+modes:
+    - scheduled
+    
+os_platform:
+    - linux
+    - windows
+    - solaris
+    - macos
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+    - Windows 10
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/file-integrity/fim-configuration.html
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html
+
+tags:
+    - syscheck
+"""
+
+import os
+import random
+import string
+import pytest
+import time
+
+from wazuh_testing.fim import (
+    LOG_FILE_PATH,
+    generate_params,
+    detect_initial_scan_start,
+    get_scan_timestamp,
+)
```

Comment:
Follow the standard defined for imports.

Reply:
Done